packages/ecs-agent/ecs.config

```diff
@@ -1,2 +1,6 @@
 ECS_LOGFILE=/var/log/ecs/ecs-agent.log
 ECS_LOGLEVEL="{{settings.ecs.loglevel}}"
+{{#if settings.network.http-proxy}}
+HTTP_PROXY={{settings.network.http-proxy}}
+{{/if}}
+NO_PROXY={{#each settings.network.no-proxy}}{{#if @index}},{{/if}}{{this}}{{/each}}{{#if settings.network.no-proxy}},{{/if}}169.254.169.254,169.254.170.2,/var/run/docker.sock
```

Comment:
Is this our proper docker socket path?

Reply:
See https://github.com/bottlerocket-os/bottlerocket/pull/1204#discussion_r521560554.